weldx/asdf/tags/weldx/core/transformations/coordinate_system_hierarchy.py

```diff
@@ -421,7 +421,7 @@ def to_tree(cls, node: CoordinateSystemManager, ctx):
         tree = {
             "name": node.name,
             "reference_time": node.reference_time,
-            "subsystem_names": sorted(subsystems),
+            "subsystem_names": subsystems,
```

Comment:
Okay, seems that order actually matters here for nested subsystems, but I think we should tackle this when we work on #354

Reply:
yes lets try this first and see later